elpy-pkg.el

```diff
@@ -5,5 +5,5 @@
                   (highlight-indentation "0.5.0")
                   (pyvenv "1.3")
                   (yasnippet "0.8.0")
-                  (s "1.11.0"))
-                )
+                  (s "1.11.0")
+                  (cl-generic "1.0"))
```

Comment:
Does this really need the newest version of cl-generic? I can't seem to find the 1.0 version for Emacs 24.5.1 Only 0.3 is available here.

It would be much appreciated if Elpy would still work Emacs 24.5.d